### ISSUE TITLE
Webpack error

### ISSUE THREAD

**Issue by djejaquino** [saucelabs/docusaurus-theme-github-codeblock#34]
Following the instructions provided in `README.md` causes webpack to throw the following error:

```
Module not found: Error: Can't resolve 'url' in '{projectPath}/node_modules/@saucelabs/theme-github-codeblock/build/theme/ReferenceCodeBlock'
BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
        - add a fallback 'resolve.fallback: { "url": require.resolve("url/") }'
        - install 'url'
If you don't want to include a polyfill, you can use an empty module like this:
        resolve.fallback: { "url": false }
client (webpack 5.75.0) compiled with 1 error
```

Apparently in order to use this theme, you need to manually configure webpack to include or ignore that `url` module. This happens because this theme [imports](https://github.com/saucelabs/docusaurus-theme-github-codeblock/blob/98d4f3d3b407be84816637e3a3f976f4fbbbe3a3/src/theme/ReferenceCodeBlock/index.tsx#L1) Node's `url` module, and [this](https://github.com/saucelabs/docusaurus-theme-github-codeblock/blob/98d4f3d3b407be84816637e3a3f976f4fbbbe3a3/src/theme/ReferenceCodeBlock/index.tsx#L36) doesn't really fix it.

Based on that, I have a few suggestions to tackle this problem:
1. Document the steps to either polyfill or ignore the `url` module;
2. Provide a plugin that does that automatically;
3. Remove dependency on `url` module;

**Comment by diemol** [comment]
Hm, then I guess this is the context I did not have from https://github.com/saucelabs/docusaurus-theme-github-codeblock/pull/26 (which we did not get in time to properly merge it).

I guess removing the dependency is what makes sense. Would you be up to send a PR for this?

**Comment by djejaquino** [comment]
@diemol Yea I can give it a shot. You guys okay with external deps or should we try to implement some of that URL logic internally?

**Comment by diemol** [comment]
I started maintaining this a few weeks ago, and I do not have a lot of context from the original implementation. From the looks of it, I would say the logic can be implemented? What do you think?

**Comment by djejaquino** [comment]
I will take a stab later. Thank

**Comment by djejaquino** [comment]
Added #35 PR
@diemol looks like just removing the import fixes the issue like #26 proposed before

**Comment by diemol** [comment]
Fixed via https://github.com/saucelabs/docusaurus-theme-github-codeblock/pull/35
